### PR TITLE
Blackrock dna

### PR DESCRIPTION
**Pull request by icelille999**
## Changelog:
- Added the rest of the Blackrock clan, (Urukal, Dal'rend, Maim, & Griselda).

## Developer changelog:
- Added Urukal, Dal'rend, Maim, & Griselda
    - Added DNA to common/dna_data/wc_orc_dna.txt
    - Added character triggers to common/scripted_triggers/wc_his_character_triggers.txt
    - Added hair scripts to gfx/portraits/portrait_modifiers/00_hairstyles_scripted_characters.txt
    - Tied DNA to historical characters in history/characters/10000_orc.txt

# How to test:

Test and give your opinion on how you think they all look in game and if they continue to look good as they age. You can find them easily attached as family to Warchief Blackhand, ruler of the Horde, in the 583 campaign. You can also test them in the other campaigns. They can be found through the "Find Character" option in the bottom menu at any time.
Note: Urukal is Blackhand's wife, while Dal'rend, Maim, and Griselda are his children.

### CODE REVIEW

**Review by TheGroxEmpire — COMMENTED** [review]
I noticed that Maim and Urukal have brown skin color. I'm not really sure if it's appropriate for them to still have brown skin color. Even the likes of Durotan and his clan clan—a clan that is far less keen about fel than the Blackhand—have mostly green color at this point due to long exposure to fel.

File: common/scripted_triggers/wc_his_character_triggers.txt [hunk 1]
```diff
@@ -129,6 +129,22 @@ is_kilrogg_trigger = {
 	exists = character:10005 # Kilrogg
```
Comment (zumbak04) [inline common/scripted_triggers/wc_his_character_triggers.txt:129]: I guess you tweaked them too much so they look like different species. Especially Urukal.
Their heads became too tall. Chin became less expressive.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/scripted_triggers/wc_his_character_triggers.txt (hunk 1 above)
Comment (icelille999) [reply common/scripted_triggers/wc_his_character_triggers.txt:129]: I mean, I get Urukal's head being tall, and I lowered the forehead height of Griselda, but I think Dal'rend and Maim look pretty close to the other orcs, especially compared to the way Blackhand looks. Can you explain what you mean?

Comment (zumbak04) [reply common/scripted_triggers/wc_his_character_triggers.txt:129]: > I lowered Urukal's head and the forehead height of Griselda in the commit I just put out, but I think Dal'rend and Maim look pretty close to the other orcs, especially compared to the way Blackhand looks. Can you explain what you mean?

After the second look, they look better than other orcs.

File: gfx/portraits/portrait_modifiers/00_hairstyles_scripted_characters.txt [hunk 2]
```diff
@@ -503,6 +503,10 @@ hairstyles_scripted_characters = {
                 add = 200
 				is_blackhand_trigger = yes
             }
+            modifier = {
+                add = 200
+				is_maim_trigger = yes
```
Comment (zumbak04) [inline gfx/portraits/portrait_modifiers/00_hairstyles_scripted_characters.txt:508]: Prob you would like to use a new hairstyle for Maim, `male_hair_indian_03_wc`. It's bald with a bun.

Comment (icelille999) [reply gfx/portraits/portrait_modifiers/00_hairstyles_scripted_characters.txt]: Updated in newest commit

**Review by zumbak04 — APPROVED** [review]
LGTM!.. but we need changes to the orcish ethnicity.

**Review by TheGroxEmpire — APPROVED** [review]
Looks like the skin color issue was addressed. They all look good to me!

**Pull request merged**